### PR TITLE
added structure for central government content

### PR DESCRIPTION
**Pull request by Abby3017**
Signed-off-by: Abby3017 <abby37kumar@gmail.com>

### CODE REVIEW

File: content/eng/central/commerce_and_industries/policy.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: "Policy"
+date: 2019-05-12T12:00:08+05:30
+draft: true
+department: "DIPTT"
```
Comment (lakshaycoder01) [inline content/eng/central/commerce_and_industries/policy.md:5]: DIPTT department dekh le

File: i18n/en.yaml [hunk 2]
```diff
@@ -54,6 +54,9 @@ license:
 poststitle:
```
Comment (lakshaycoder01) [inline i18n/en.yaml:54]: T ya Caps rakh ya space de

File: content/eng/central/commerce_and_industries/policy.md (hunk 1 above)
Comment (lakshaycoder01) [reply content/eng/central/commerce_and_industries/policy.md:5]: Change Department to Ministeries

File: i18n/en.yaml (hunk 2 above)
Comment (Abby3017) [reply i18n/en.yaml:54]: this is format in yaml. lets continue that for now

**Pull request merged**